docs/en/remote-flow.rst

```diff
@@ -60,7 +61,7 @@ The details of each step shown in the previous picture are described in the tabl
   * - **13**
     - When the Wallet Instance capabilities discovery is not supported by RP, the Wallet Instance request the signed Request Object using the HTTP method GET.
   * - **14**
-    - The Wallet Instance obtains the signed Request Object.
+    - The RP issues the Request Object signin it using one of its cryptographic private keys, where their public parts have been published within its Entity Configuration (`metadata.openid_wallet_relying_party.jwks`). The Wallet Instance obtains the signed Request Object.
```

Comment:
Secondo le specs correnti, i metadata dell'RP sono in `metadata.wallet_relying_party.jwks`. non, `metadata.openid_wallet_relying_party.jwks` (si osservi l'omissione del prefisso `openid_`). Quale fa fede?

Reply:
decisamente wallet_relying_party